astroquery/mast/cutouts.py

```diff
@@ -70,21 +71,37 @@ def _parse_cutout_size(size):
                       InputWarning)
 
     # Getting x and y out of the size
+
     if np.isscalar(size[0]):
         x = size[1]
         y = size[0]
         units = "px"
+        limit_reached = True if ((x >= 30) or (y >= 30)) else False
+
     elif size[0].unit == u.pixel:
         x = size[1].value
         y = size[0].value
         units = "px"
+        limit_reached = True if ((x >= 30) or (y >= 30)) else False
+
     elif size[0].unit.physical_type == 'angle':
         x = size[1].to(u.deg).value
         y = size[0].to(u.deg).value
         units = "d"
+
+        # Based on the literature, TESS resolution is approx. 21 arcseconds per pixel.
+        # We will convert the recommended upper limit for a dimension from pixels
+        # to degrees.
+        upper_limit = (30 * 21*u.arcsec).to(u.deg)
```

Comment:
Why is the conversion to degrees needed?
```suggestion
        upper_limit = 30 * 21*u.arcsec
```

Reply:
Because the code needs to represent the upper limit in terms of degrees for the cases where users input their cutout size in degrees rather than pixels.